README.md

```diff
@@ -497,6 +497,7 @@ than one leads to undefined behaviour).
  * Detect virtualised hosts.
  * Unrestrict the dmesg buffer (Linux Kernel 4.8+)
  * Turn off "turbo boost" (Linux only)
+ * Set the memory over-commit policy (Linux only).
```

Comment:
Don't we need to say "turns off memory over-commit" or similar? i.e. tell people *what* policy we're choosing?

Reply:
Fixed. Try that.